.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+[Please provide general introduction to the issue/proposal.]
+
+[If reporting a bug, attach also the entire traceback from Python. If proposing an enhancement/new feature, provide links to the related articles, reference examples, etc.]
+
+
+## Steps to reproduce
+[If possible, please include example code and image(s) to reproduce the issue/new results.]
```

Comment:
Not much. It is supposed to facilitate work for developers who like to "play" with the code first.